### DISCUSSION TITLE
Photo export for sharing

### DISCUSSION THREAD

**Discussion by blue-kaleidoscope** [immich-app/immich#10525] [General]
Hello friendly immich people,

I hope this is not a double request, at least I couldn't find anything similar in the discussions: I was wondering whether Immich supports an image export feature similar to Lightroom (but less complex) which allows:
- Download one or multiple images (as today) but in a different format
- In a different size.

The use case behind: Yesterday I wanted to send a friend some ~20 pictures via mail. I downloaded the selection via my immich instance. As these were iPhone photos I ended up with 50 MB of HEIF format photos which I needed to convert to jpeg with a lower resolution so that they fitted into the mail.

Is something like that planned?

Many thanks,
Oli

**Comment by bo0tzz** [comment]
Immich doesn't currently have any way to export files in different images. Would just a normal shared album link work for what you need?

**Reply by blue-kaleidoscope** [reply]
Not really as I do not want to expose my instance to the evil internetz. ^^

**Comment by Hej-world** [comment]
That would indeed be a great feature as I normally upload photos from Lightroom and they are done with a DSLR.

Would it be possible to add a download option for the already created thumbnails? These are smaller file size and perfect to quickly send via e.g. WhatsApp.

Many thanks!

**Reply by NicholasFlamy** [reply]
> Would it be possible to add a download option for the already created thumbnails?

This is related to #6723
